documentation/library_guide/dynamic_selection_api/auto_tune_policy.rst

```diff
@@ -175,6 +175,12 @@ Constructors
   * - ``auto_tune_policy(const std::vector<resource_type>& u, uint64_t resample_interval_in_milliseconds = 0);``
     - Overrides the default set of resources. An optional resampling interval can be provided.
 
+.. Note::
+
+   When initializing the ``auto_tune_policy`` with SYCL queues, constructed with the

```

Comment:
```suggestion
   When initializing the ``auto_tune_policy`` with SYCL queues, constructing the queues with the ``sycl::property::queue::enable_profiling`` property allows a more accurate determination of the best-performing device to be made.

```

Reply:
I'm concerned that "using the timing data" would lead a reader to think that they need to use the timing data in their code.  The timing data is used within the auto_tune_policy.